.github/workflows/build_and_test_dbt.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
         uses: ./.github/actions/setup_dbt
         with:
           role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
+          role-duration-seconds: 14400  # Worst-case time for full build
```

Comment:
I debated storing this value as a repo secret, but decided against it since it doesn't feel to me like it makes the role any less secure. Still, if you disagree I'd be happy to go ahead and edit the history to remove this value and move it to a repo secret.